examples/scripts/sentiment_tuning.py

```diff
@@ -56,6 +55,9 @@ class ScriptArguments:
     )
     target_kl: Optional[float] = field(default=0.1, metadata={"help": "kl target for early stopping"})
     seed: Optional[int] = field(default=0, metadata={"help": "the random seed"})
+    use_score_scaling: Optional[bool] = field(default=False, metadata={"help": "Use score scaling"})
+    use_score_norm: Optional[bool] = field(default=False, metadata={"help": "Use score normalization"})
```

Comment:
Maybe we should clarify that this only works if `use_score_scaling` is also `True` otherwise it's actually ignored. we change the logic a bit in general

Reply:
Done